Update README.md (#28)
dfinity/sns-testing: README.md

   for further details) which can be thought of as a placeholder
   for your dapp.

> 6. Run the script

Author: ```suggestion
5. Run the script
```

   Note that it might take a few minutes before the change is propagated to this website).
   
8. Run the script
   ```bash
   ./participate_sns_swap.sh <num-participants> <icp-per-participant>  # from Bash
   ``` 
   to participate in the swap, providing the number of
   participants and the number of ICP that each participant contributes as arguments.

   You can run the script `./participate_sns_swap.sh` multiple times as long as
   the sum of provided `<icp-per-participant>` does not exceed the maximum amount of ICP
   per participant (specified in the NNS proposal to open the swap).

   You can also participate in the swap using the [NNS frontend dapp](http://qsgjb-riaaa-aaaaa-aaaga-cai.localhost:8080/) and the "Get ICP" button in the [NNS frontend dapp](http://qsgjb-riaaa-aaaaa-aaaga-cai.localhost:8080/) to get a sufficient amount of ICP.

   Make sure that the participation satisfies all the constraints
   imposed by the swap parameters from the previous step (e.g., the minimum number
   of swap participants and the total amount of ICP raised). For example,
   to contribute the Minimum Commitment to the swap right away, run: 

   ```bash

>    ./participate_sns_sale.sh 4 12

Author: ```suggestion
   ./participate_sns_sale.sh 5 100
```

   ```

> 

Author: ```suggestion
```

   ``` 
   to open the initial decentralization swap.
   You can adjust the swap parameters directly in the script.

   After this step, you should see a new SNS instance in the [Launchpad](http://qsgjb-riaaa-aaaaa-aaaga-cai.localhost:8080/launchpad/).

>    Note that it might take a few minutes before the change is propagated to this website).

Reviewer: ```suggestion
   (Note that it might take a few minutes before the change is propagated to this website).
```

   
6. Run the script
   ```bash
   ./participate_sns_swap.sh <num-participants> <icp-per-participant>  # from Bash
   ``` 
   to participate in the swap, providing the number of
   participants and the number of ICP that each participant contributes as arguments.

   You can run the script `./participate_sns_swap.sh` multiple times as long as
   the sum of provided `<icp-per-participant>` does not exceed the maximum amount of ICP
   per participant (specified in the NNS proposal to open the swap).

   You can also participate in the swap using the [NNS frontend dapp](http://qsgjb-riaaa-aaaaa-aaaga-cai.localhost:8080/) and the "Get ICP" button in the [NNS frontend dapp](http://qsgjb-riaaa-aaaaa-aaaga-cai.localhost:8080/) to get a sufficient amount of ICP.

   Make sure that the participation satisfies all the constraints
   imposed by the swap parameters from the previous step (e.g., the minimum number
   of swap participants and the total amount of ICP raised). For example,

>    to contribute the Minimum Commitment to the swap right away, run: 

Author: ```suggestion
   to contribute enough ICP to be able to finalize the swap right away, run:
```

   ```bash

>    ./participate_sns_sale.sh 5 100

Author: ```suggestion
   ./participate_sns_sale.sh 3 10
```

   or the participation is insufficient (too few participants, minimum amount of ICP not reached),
   then the finalization is expected to fail.

> 11. Upgrade your dapp again by submitting an SNS proposal that can be voted on using the SNS developer neuron. This however might not be enough to execute the upgrade, so you also need to vote on this proposal using your participants' neurons (this will be covered in the next step).

Author: ```suggestion
8. Upgrade your dapp again by submitting an SNS proposal that can be voted on using the SNS developer neuron. This however might not be enough to execute the upgrade, so you also need to vote on this proposal using your participants' neurons (this will be covered in the next step).
```

   for further details) to use a new greeting when calling the `greet` method it exposes. If you don't provide `<new_greeting>`, `"Hoi"` will be used by default. 
   The test canister can be thought of as a placeholder for your dapp.

> 11. After the decentralization swap, your developer neuron might not have

Author: ```suggestion
9. After the decentralization swap, your developer neuron might not have
```
